rotkehlchen/api/rest.py

```diff
@@ -4752,3 +4755,27 @@ def get_spam_assets_false_positives(self) -> Response:
             )
 
         return api_response(_wrap_in_ok_result(whitelisted_tokens), status_code=HTTPStatus.OK)
+
+    def add_to_spam(self, token: EvmToken) -> Response:
```

Comment:
1. Rename
2. Add a docstring

```suggestion
    def add_token_to_spam(self, token: EvmToken) -> Response:
```